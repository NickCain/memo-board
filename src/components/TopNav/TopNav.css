.TopNav {
    width: 100%;
    padding: 1rem;
    display: flex;
    flex-flow: row wrap;
}

.TopNav .title {
    margin: 0 10px;
}
