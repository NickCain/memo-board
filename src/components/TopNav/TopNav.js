import PropTypes from 'prop-types';
import React, { Component } from 'react';

import Button from '../../components/common/Button/Button';

import './TopNav.css';

class TopNav extends Component {
  constructor(props) {
    super(props);

    this.state = {};
  }

  render() {
    const { btnText, onBtnClick } = this.props;
    return (
      <div className="TopNav">
        <h1 className="title"> Memo board</h1>
        <Button text={btnText} onClick={onBtnClick}/>
      </div>
    );
  }
}

TopNav.propTypes = {
  onBtnClick: PropTypes.func,
  btnText: PropTypes.string
};

export default TopNav;
