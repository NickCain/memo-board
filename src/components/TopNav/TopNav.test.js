import { mount, shallow } from 'enzyme';
import React from 'react';
import TopNav from './TopNav.js'

let wrapper;
let shallowWrapper;

describe('TopNav - component', () => {
  describe('TopNav renders', () => {
    beforeEach(() => {
      wrapper = mount(
        <TopNav />
      );
      shallowWrapper = shallow(
        <TopNav />
      );
    });

    it('should render the TopNav component', () => {
      expect(shallowWrapper).toMatchSnapshot();
      expect(wrapper.length).toBeGreaterThan(0);
    });
  });
});
