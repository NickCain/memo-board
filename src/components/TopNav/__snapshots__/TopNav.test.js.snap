// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopNav - component TopNav renders should render the TopNav component 1`] = `undefined`;
